ios/FluentUI/TextField/FluentTextField.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import Combine
+
+@objc(MSFTextField)
+public final class FluentTextField: UIView, UITextFieldDelegate, TokenizedControlInternal {
+    public override func willMove(toWindow newWindow: UIWindow?) {
+        super.willMove(toWindow: newWindow)
+        guard let newWindow else {
+            return
+        }
+        tokenSet.update(newWindow.fluentTheme)
+        updateTokenizedValues()
+    }
+
+    public typealias TokenSetKeyType = TextFieldTokenSet.Tokens
+    lazy public var tokenSet: TextFieldTokenSet = .init(state: { [weak self] in
+        return self?.state ?? .unfocused
+    })
+
+    /// UIImage used in the leading UIImageView. If this is nil, the leading UIImageView will be hidden.
+    @objc public var leadingImage: UIImage? {
```

Comment:
at any point, how would client set accessibility text on this icon?

Reply:
The leading icon isn't currently accessible. VoiceOver only goes over the title, input/placeholder, clear button, and assistive text. Leading images don't generally seem to be accessible in our other controls, either, like Button, Notification, and TableViewCell.